include/index.js

```diff
@@ -211,9 +214,8 @@ module.exports = function PB(config) {
     pb.AdminSubnavService = require(path.join(config.docRoot, '/include/service/admin/admin_subnav_service.js'))(pb);
     pb.AnalyticsManager   = require(path.join(config.docRoot, '/include/system/analytics_manager.js'))(pb);
     pb.UrlService         = require(path.join(config.docRoot, '/include/service/entities/url_service.js'))(pb);
-    pb.CallHomeService    = require(path.join(config.docRoot, '/include/system/call_home_service.js'))(pb);
     pb.JobService         = require(path.join(config.docRoot, '/include/service/entities/job_service.js'))(pb);
-    pb.TokenService       = require(path.join(config.docRoot, '/include/service/entities/token_service.js'))(pb);
+    pb.TokenServiceV2     = require(path.join(config.docRoot, '/include/koa/authentication/token_service.js'))(pb);
```

Comment:
If we remove the TokenService, we need to edit the TokenServiceModule. That is the one that manage the session (connect to another site when you're logged in global). I tested it out right now, and it throws the following error:

`2018-11-16T22:17:36.862Z - error: [M]  TypeError: pb.TokenService is not a constructor
    at AuthTokenControllerModule.AuthTokenController.render (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/plugins/pencilblue/controllers/actions/admin/sites/auth_token.js:40:28)
    at Promise (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/include/http/middleware/controller.js:27:48)
    at Promise._execute (/Users/dacm26/Documents/Projects/Number8/career-builder/CMSPencilblue/node_modules/bluebird/js/release/debuggability.js:303:9)
    at Promise._resolveFromExecutor (/Users/dacm26/Documents/Projects/Number8/career-builder/CMSPencilblue/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/Users/dacm26/Documents/Projects/Number8/career-builder/CMSPencilblue/node_modules/bluebird/js/release/promise.js:79:10)
    at render (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/include/http/middleware/controller.js:26:29)
    at dispatch (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at setSecureResponseHeaders (/Users/dacm26/Documents/Projects/Number8/career-builder/CMSPencilblue/plugins/cms_base/middleware/index.js:78:20)
    at dispatch (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/node_modules/koa-router/node_modules/koa-compose/index.js:44:32)
    at next (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/node_modules/koa-router/node_modules/koa-compose/index.js:45:18)
    at initializeController (/Users/dacm26/Documents/Projects/Number8/career-builder/pencilblue/include/http/middleware/controller.js:19:15)
    at <anonymous>:null:null`